### PR TITLE
Improve chat-plugins

### PR DESCRIPTION
**Pull request by Bryan-0**

### CODE REVIEW

**Comment by Slayer95** [comment]
> Replace this.sendReply with this.errorReply in some cases.

Make this the commit message (also add ``chat-plugins``)

**Comment by Zarel** [comment]
Using <code>``</code> is PS convention, just use <code>`</code> everywhere else.

**Comment by Bryan-0** [comment]
This is fine? @Zarel  @Slayer95

File: chat-plugins/scavengers.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ exports.commands = {
 	},
 	startofficialhunt: 'starthunt',
 	starthunt: function (target, room, user, connection, cmd) {
-		if (room.id !== 'scavengers') return this.sendReply('This command can only be used in the Scavengers room.');
+		if (room.id !== 'scavengers') return this.errorReply('This command can only be used in the Scavengers room.');
 		if (!this.can('mute', null, room)) return false;
 		if (scavengers.status === 'on') return this.sendReply('There is already an active scavenger hunt.');
```
Comment (panpawn) [inline chat-plugins/scavengers.js]: What about these kinds of checks?  Should they also use ``this.errorReply()``?

Comment (Bryan-0) [inline chat-plugins/scavengers.js]: I was thinking that, but I'm not sure.

File: logs/repl/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# repl directory
+# Repl directory
```
Comment (Zarel) [inline logs/repl/README.md]: The directory is named `repl`, not `Repl`. Directory names are case sensitive in many operating systems, so this is relevant.

Perhaps it'd be clearer as <code># `logs/repl/` directory</code>

File: logs/repl/README.md [hunk 3]
```diff
@@ -6,4 +6,4 @@ This directory is used by default to store the REPL sockets to all of showdown's
 
 The intended uses of these REPL sockets are for debugging (especially when the server is seemingly frozen) and scripting.
 
-You can use any tool capable of talking to Unix sockets such as `nc`. e.g., `nc -U app`
\ No newline at end of file
+You can use any tool capable of talking to Unix sockets such as `nc`. e.g., `nc -U app`
```
Comment (Zarel) [inline logs/repl/README.md]: I have no idea what you changed here, but you might as well remove the comma after `e.g.` if you're fixing grammar.

Comment (Bryan-0) [inline logs/repl/README.md]: I didn't touch that, but I'll fix it.

Comment (Zarel) [inline logs/repl/README.md]: Diff says you did. Maybe you changed the line endings? That'd be bad.

File: chat-plugins/scavengers.js (hunk 1 above)
Comment (Zarel) [inline chat-plugins/scavengers.js]: If you'd like.

**Comment by Bryan-0** [comment]
Ok, I'm going to update the files.

**Comment by Bryan-0** [comment]
Damn it, fail -_-

**Comment by Bryan-0** [comment]
Updated. @Zarel

**Pull request merged**